app/controllers/api/v1/rides_controller.rb

```diff
@@ -5,8 +5,8 @@ def show
       amusement_park_id: AmusementPark.find(params[:amusement_park_id]).id,
       ride: Ride.find(params[:id]),
       features: serialized_features,
-      ride_reviews: serialized_ride_reviews
-      }
+      ride_review: serialized_ride_reviews
```

Comment:
Is there a reason you made this singular? Looks like it's still an array so I would re-pluralize